src/palace/manager/service/storage/s3.py

```diff
@@ -244,3 +244,60 @@ def multipart_abort(self, key: str, upload_id: str) -> None:
             Key=key,
             UploadId=upload_id,
         )
+
+    def update_bucket_expiration_rule(
+        self, prefix: str, expiration_in_days: int = 30
+    ) -> None:
```

Comment:
Minor: Seems like this might be something we want to just do in our terraform config, instead of doing it every time we do an inventory report. We already apply a lifecycle rule to [expire multipart uploads](https://github.com/ThePalaceProject/hosting-playbook/blob/main/terraform/modules/s3/main.tf#L71-L91).

Reply:
👍